dvc/utils.py

```diff
@@ -165,3 +167,19 @@ def fix_env(env):
             env.pop(lp_key, None)
 
     return env
+
+
+def visual_width(line):
+    """ Get the the number of columns required to display a string """
+
+    return len(re.sub(colorama.ansitowin32.AnsiToWin32.ANSI_CSI_RE, '', line))
```

Comment:
I don't think we need re.sub here, `line.replace()` should be enough 🙂

Reply:
Sorry, my mistake. ANSI_CSI_RE is a true regex, so we do need re.sub.

Reply:
I'm afraid we do, @efiop , `str.replace(old, new)` `old` needs to be a string, and we are using a compiled regex from colorama